apps/server/src/utils/__tests__/getFileListFromFolder.test.ts

```diff
@@ -0,0 +1,34 @@
+import { expect, vi } from 'vitest';
+
+import { getProjectFiles } from '../getFileListFromFolder.js';
+
+describe('getProjectFiles test', async () => {
+  it('should return a list of project .json files', async () => {
```

Comment:
I am not sure mocking the fs stuff provides much value. Perhaps we take advantage of the structure and test only the filter part?

Reply:
fair, honestly with this I'm only testing the filtering anyway since mocking the `fs` mocking doesn't help to test the `fs` part